FND/Utilities/Analytics.rdf

```diff
@@ -101,6 +101,14 @@
 		<owl:equivalentClass rdf:resource="&fibo-fnd-utl-alx;Ratio"/>
 	</owl:Class>
 	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;characterizes">
+		<rdfs:domain rdf:resource="&fibo-fnd-utl-alx;Aspect"/>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;isCharacterizedBy">
+		<rdfs:range rdf:resource="&fibo-fnd-utl-alx;Aspect"/>
```

Comment:
we have AddressComponent as a subclass of Aspect, which I think is wrong: these restrictions do not work at all well for that.

Reply:
fixed - made it a constituent instead